src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0 Added support for OVN
+
+> This release mainly focused on adding support for interconnecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+
+Other additions and changes of note:
```

Comment:
```suggestion
```

Reply:
We don't say this for the other sections, let's not say it here imho.

Reply:
yeah but maybe we should? It seems we describe a main feature first in a separate font/color then list the remaining items. Or maybe we should just not summarize a main feature and just list it with the rest.  I kinda prefer that.

## v0.8.0

* main item
* ...

Reply:
Yeah I'd prefer not to try to highlight one "main" change, I don't think that is very accurate and I don't find it useful for making other communications (Tweets).

Reply:
agreed